templates/expo-template-default/app.json

```diff
@@ -13,21 +13,21 @@
     },
     "android": {
       "adaptiveIcon": {
-        "foregroundImage": "./assets/images/adaptive-icon.png",
+        "foregroundImage": "./src/assets/images/adaptive-icon.png",
```

Comment:
Is it common to include assets in the src folder?

Reply:
I don't think so, that was an oversight.